scala/sag_12r.par

```diff
@@ -1,4 +1,4 @@
-! Sagittal 12R notation. Dave Keenan & George Secor. 2006-Mar-22, updated 2020-Apr-14
+! Sagittal 12R (Trojan) notation. Dave Keenan & George Secor. 2006-Mar-22, updated 2020-Jun-14
```

Comment:
I thought we agreed to "12 relative" rather than "12R".

Reply:
Also good catch. Thanks. According to http://forum.sagittal.org/viewtopic.php?p=1594#p1594 it looks like the final form was "12-Relative".

Reply:
Agreed. Edited my comment above.